mmv1/third_party/terraform/resources/resource_storage_transfer_job.go

```diff
@@ -944,12 +949,14 @@ func expandAwsS3Data(awsS3Datas []interface{}) *storagetransfer.AwsS3Data {
 		BucketName:   awsS3Data["bucket_name"].(string),
 		AwsAccessKey: expandAwsAccessKeys(awsS3Data["aws_access_key"].([]interface{})),
 		RoleArn:      awsS3Data["role_arn"].(string),
+		Path:         awsS3Data["path"].(string),
 	}
 }
 
 func flattenAwsS3Data(awsS3Data *storagetransfer.AwsS3Data, d *schema.ResourceData) []map[string]interface{} {
 	data := map[string]interface{}{
 		"bucket_name": awsS3Data.BucketName,
+		"path": awsS3Data.Path,
```

Comment:
The checks failed. `awsS3Data.Path` should be aligned with other values, like `awsS3Data.BucketName`